packages/cli/lib/utils.js

```diff
@@ -370,3 +370,24 @@ function validateValue(name, unallowedCharacters) {
   }
   return true;
 }
+/**
+ *  Returns the modelName in the directory file format for the model
+ * @param {string} modelName
+ */
+exports.toModelFileName = function(modelName) {
```

Comment:
Perhaps `getModelFileName` would be a better function name?

Reply:
There was method called toClassName already in utils.js , so I wanted to keep the same line with toModelFileName. But I believe we can change here toModelFileName and then probably open another PR to change the current one toClassName .